### PR TITLE
[RFR] Added “React” entry

### PR DESCRIPTION
**Pull request by CrocoDillon**
Because React deserves to be completed sooner than later :shipit:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,13 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
 
 ## React
 
-[React](https://facebook.github.io/react/) section to be completed.
+[React](https://facebook.github.io/react/) is a library developed and used at Facebook for building user interfaces. It can be seen as the V in MVC as it makes no assumptions about the rest of the technology stack. Using React Native it can even be used to power native apps.
```
Comment (KittyGiraudel) [inline README.md]: Add _(Model View Controller)_ after _MVC_.
Maybe a link to React Native? Would be interesting. To an entry of ours if we have one, or to a relevant resource.

File: README.md [hunk 2]
```diff
@@ -112,7 +112,13 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
...
+
+In React you can write HTML directly in JS using an XML-like syntax called JSX. JSX compiles to JS and is optional, but does make the code more expressive.
+
+Data flow in React is one-way which makes it easier to reason about and avoid mistakes. This quality can be enhanced using Flux, Facebook’s complementary application architecture, or Redux which many people see as a “better Flux”.
```
Comment (KittyGiraudel) [inline README.md]: Making _Flux_ a link to `#flux` would be nice. :)
Making _Redux_ a link to `#redux` would be nice. :)

File: README.md [hunk 3]
```diff
@@ -112,7 +112,13 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
...
+
+React is not a framework.
```
Comment (KittyGiraudel) [inline README.md]: Kind of weird to finish with is. I would have actually said it is a framework as it kind of enforces a way of doing things (in a way). :x

Comment (CrocoDillon) [inline README.md]: Hahaha, it’s a lame joke that circulated on social media a while ago. For example:

react-no-framework

I'll remove it.

Comment (CrocoDillon) [inline README.md]: Here, if you want more: https://twitter.com/jaffathecake/status/606064895805792256

**Comment by KittyGiraudel** [comment]
Awesome! :)

**Comment by CrocoDillon** [comment]
Glad you like it!

**Pull request merged**